python/arcticdb/version_store/library.py

```diff
@@ -1223,16 +1228,18 @@ def write_metadata(self, symbol: str, metadata: Any) -> VersionedItem:
             Symbol name for the item
         metadata
             Metadata to persist along with the symbol
+        prune_previous_versions : `Optional[bool]`, default=False
+            Removes previous (non-snapshotted) versions from the database.
```

Comment:
Can we make it clear, perhaps warn, that this removes previous data, not just metadata.